.github/workflows/triage.md

```diff
@@ -180,14 +180,21 @@ Analyze and determine:
 
 ---
 
-## Wrong Repository
+## Recommendation for Triage
+
+**This issue appears to belong in a different repository.**
```

Comment:
This is a bit nitpicky and might be better decided with user testing, but I think even this wording might be a bit forward.
```suggestion
**This issue might be better filed in a different repository.**
```